projects/igniteui-angular/migrations/migration-collection.json

```diff
@@ -75,6 +75,11 @@
             "version": "9.1.0",
             "description": "Updates Ignite UI for Angular from v9.0.x to v9.1.0",
             "factory": "./update-9_1_0"
+        },
+        "migration-16": {
+            "version": "10.1.0",
+            "description": "Updates Ignite UI for Angular from v9.1.x to v10.1.0",
```

Comment:
I guess this can be from `10.0.x` to 10.1.0 (though we don't technically have a migration for 10.0.0)